.github/workflows/dockerhub.yml

```diff
@@ -1,25 +0,0 @@
-name: DockerHub
```

Comment:
This CI did not make any sense. It was pushing a docker image without tag information about which config it was.

Reply:
agree